.env.example

```diff
@@ -34,3 +34,6 @@ NEXTAUTH_URL="http://localhost:3000"
 # Next Auth Discord Provider (lmao)
 DISCORD_CLIENT_ID=
 DISCORD_CLIENT_SECRET=
+
+# Next auth secret
+NEXT_AUTH_SECRET=1203jdifjmkldsmldas
```

Comment:
Is it possible to remove this secret? You can share it with us through private channels.

Reply:
It's just a random secret for jwt hashing, the text here is just an example, you can set it to what ever you like on your local dotenv

Reply:
I'll switch it to something more meaningful